Purchases/Purchasing/StoreKitAbstractions/SubscriptionPeriod.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  SubscriptionPeriod.swift
+//
+//  Created by Andrés Boedo on 3/12/21.
+
+import Foundation
+import StoreKit
+
+@objc public class SubscriptionPeriod: NSObject {
+
+    public let value: Int
+    public let unit: PeriodUnit
+
+    init(value: Int, unit: PeriodUnit) {
+        self.value = value
+        self.unit = unit
+    }
+
+    @objc public enum PeriodUnit: Int {
```

Comment:
I like this abstraction. As a user I'd rather use RevenutCat types over StoreKit types.